vignettes/using_lintr.Rmd

```diff
@@ -0,0 +1,311 @@
+---
+title: "Using lintr"
+author: "Alexander Rosenstock"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Using lintr}
+  %\VignetteEngine{knitr::rmarkdown}
+  \usepackage[utf8]{inputenc}
+---
+
+This vignette describes how to set up and configure `lintr` for use with projects or packages.
+
+## Running `lintr` on a project
```

Comment:
maybe a better title for this section: "Running `lintr` at different granularities" or "Different levels of running `lintr` on a project"

Reply:
Also it's probably useful to emphasize the sequence -- `lint_dir` calls `lint`, and `lint_package` calls `lint_dir` internally/